package.json

```diff
@@ -7,8 +7,8 @@
     "url": "git+https://github.com/xmartlabs/xmartlabs-react-template.git"
   },
   "engines": {
-    "node": "20.8.1",
-    "npm": "10.1.0"
+    "node": "22.8.0",
+    "npm": "10.8.2"
```

Comment:
This version does not match with the one in the Readme file

Reply:
Good catch!